### ISSUE TITLE
Cannot return null for non-nullable field Query.cardano.

### ISSUE THREAD

**Issue by petarmitrovic** [cardano-foundation/cardano-graphql#906]
### Summary

After upgrading our Cardano stack to the latest versions I'm getting the following error when querying the latest block:
```
curl \
  -X POST \
  -H "Content-Type: application/json" \
  -d '{"query": "{ cardano { tip { number slotNo epoch { number } } } }"}' http://localhost:3100/graphql
{"errors":[{"message":"Cannot return null for non-nullable field Query.cardano.","locations":[{"line":1,"column":3}],"path":["cardano"],"extensions":{"code":"INTERNAL_SERVER_ERROR","exception":{"stacktrace":["Error: Cannot return null for non-nullable field Query.cardano.","    at completeValue (/app/node_modules/graphql/execution/execute.js:560:13)","    at /app/node_modules/graphql/execution/execute.js:492:16","    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","    at async Promise.all (index 0)"]}}}],"data":null}
```

I have updated the stack according to the [compatibility matrix](https://docs.cardano.org/developer-resources/release-notes/comp-matrix/):
* cardano-node from 9.2.0 to 10.1.2
* cardano-db-sync from 13.5.0.2 to 13.6.0.2
* cardano-graphql from 8.2.2 to 8.3.0

I didn't find any particular instruction in release notes about required schema upgrade.

In the graphql logs I see the following:
```
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":141743419,"id":"8942d748f30ba4aaa3786eec29c79735f625611aefb1218ca25993786d000114"},"msg":"[Object: null prototype] {\n  slot: 141743419,\n  id: '8942d748f30ba4aaa3786eec29c79735f625611aefb1218ca25993786d000114'\n}","time":"2024-12-04T10:55:20.518Z","v":0}
```

### Steps to reproduce the bug

Apply upgrade of the components as described above and try fetching the latest block from the node:
```
curl \
  -X POST \
  -H "Content-Type: application/json" \
  -d '{"query": "{ cardano { tip { number slotNo epoch { number } } } }"}' http://localhost:3100/graphql
```

### Actual Result

The response received is:
```
{"errors":[{"message":"Cannot return null for non-nullable field Query.cardano.","locations":[{"line":1,"column":3}],"path":["cardano"],"extensions":{"code":"INTERNAL_SERVER_ERROR","exception":{"stacktrace":["Error: Cannot return null for non-nullable field Query.cardano.","    at completeValue (/app/node_modules/graphql/execution/execute.js:560:13)","    at /app/node_modules/graphql/execution/execute.js:492:16","    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","    at async Promise.all (index 0)"]}}}],"data":null}
```

### Expected Result

In the previous version the node used to return the following response:
```
{"data":{"cardano":{"tip":{"number":11176239,"slotNo":"141739470","epoch":{"number":525}}}}}
```

### Environment

Updated to 8.3.0.

### Platform

- [ ] Linux (Ubuntu)
- [ ] Linux (Other)
- [ ] macOS
- [ ] Windows

### Runtime

- [ ] Node.js
- [X] Docker

**Comment by Kammerlo** [comment]
Hey, Sorry I can't reproduce this bug on my environment. A few questions:
- Could you check if `db-sync` is running fine? If not could you provide logs
- Could you check if the `server` and `background` component is logging anything suspicious?

**Comment by petarmitrovic** [comment]
Thanks for looking into this.

I don't see anything suspicious in the logs. Here are the latest lines from `db-sync`:
```
[db-sync-node:Info:84] [2024-12-09 14:55:26.01 UTC] Insert Conway Block: epoch 526, slot 142189834, block 11197990, hash b666da264bc23916a0cdfa6cd9bdf254e4fda3361366ff676e4a3bd049389433
[db-sync-node:Info:84] [2024-12-09 14:55:39.69 UTC] Insert Conway Block: epoch 526, slot 142189848, block 11197991, hash 4499b5381fdf147c1ad86d94efb713a7822aeb872584bde0c005f006f2d771ef
[db-sync-node:Info:84] [2024-12-09 14:56:47.49 UTC] Insert Conway Block: epoch 526, slot 142189913, block 11197992, hash abedd6a89cc289bc0d20cd32fb48ebc043f897b58432b894288097f4da4e7455
[db-sync-node:Info:84] [2024-12-09 14:56:55.64 UTC] Insert Conway Block: epoch 526, slot 142189924, block 11197993, hash 75a59fa31475f4b12e0e44e2932e2572c16364a203e28e0471a656f308edb636
[db-sync-node:Info:84] [2024-12-09 14:57:18.54 UTC] Insert Conway Block: epoch 526, slot 142189946, block 11197994, hash 4e1d4b57692af3a4e8372047a047b849376d86ab801efc8954198d96fe2faab3
[db-sync-node:Info:84] [2024-12-09 14:57:45.76 UTC] Insert Conway Block: epoch 526, slot 142189973, block 11197995, hash 86506254144a2bc00cd67ba2be8aa509625ed4ea2d7a184a43db5a11440e4d6a
[db-sync-node:Info:84] [2024-12-09 14:57:57.92 UTC] Insert Conway Block: epoch 526, slot 142189986, block 11197996, hash c9a9177c9bd9bf7b07bd7e5030bfd4dcebef10a508960ab31d4e3f7df06b04ba
[db-sync-node:Info:84] [2024-12-09 14:58:16.11 UTC] Insert Conway Block: epoch 526, slot 142190004, block 11197997, hash 8474d90e5000d75ddafed49628d68ad6edf5a5e03025ef577fc17ac2336d07e5
[db-sync-node:Info:84] [2024-12-09 14:58:42.31 UTC] Insert Conway Block: epoch 526, slot 142190030, block 11197998, hash a5b9fb717feeae690e8ff3b10c7695b1f6a66f4a30a66e8cf3339bb74195ac57
[db-sync-node:Info:84] [2024-12-09 15:00:13.27 UTC] Insert Conway Block: epoch 526, slot 142190120, block 11197999, hash fd5c61fe16e630a55e39324eeaf7ac5ed0315ac3d66a99b5939e7934b99121c0
Logs from 09/12/2024, 13:49:32
```

For the node I'm seeing some warnings I'm not quite sure what they are about:
```
[cardano-:cardano.node.ChainDB:Notice:26511] [2024-12-09 15:01:49.82 UTC] Chain extended, new tip: cfd813ae3430ff970beec8df41c2758b73e263a98cef49d8e3126e3c56ff424e at slot 142190218
[cardano-:cardano.node.Mempool:Info:26519] [2024-12-09 15:01:49.82 UTC] fromList [("enclosingTime",Object (fromList [("tag",String "RisingEdge")])),("kind",String "TraceMempoolSynced")]
[cardano-:cardano.node.Mempool:Info:26519] [2024-12-09 15:01:49.82 UTC] fromList [("enclosingTime",Object (fromList [("contents",Number 8.29281e-4),("tag",String "FallingEdgeWith")])),("kind",String "TraceMempoolSynced")]
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:01:54.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:54642 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 33> closed when reading data, waiting on next header True"))) 1s 20s
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:01:59.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:35482 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 42> closed when reading data, waiting on next header True"))) 1s 20s
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:02:04.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:35494 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 47> closed when reading data, waiting on next header True"))) 1s 20s
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:02:09.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:59384 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 52> closed when reading data, waiting on next header True"))) 1s 20s
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:02:14.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:59390 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 57> closed when reading data, waiting on next header True"))) 1s 20s
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:02:19.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:40192 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 62> closed when reading data, waiting on next header True"))) 1s 20s
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:02:24.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:40196 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 67> closed when reading data, waiting on next header True"))) 1s 20s
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:02:29.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:36950 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 72> closed when reading data, waiting on next header True"))) 1s 20s
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:02:34.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:36956 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 77> closed when reading data, waiting on next header True"))) 1s 20s
[cardano-:cardano.node.ErrorPolicy:Warning:26563] [2024-12-09 15:02:39.72 UTC] IP [2a01:4f8:fff0:9e:1::de68]:54634 ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed "<socket: 82> closed when reading data, waiting on next header True"))) 1s 20s
```

For graphql (server):
```
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:10.385Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191193,"id":"ff3710a432d599f379c38fc110d8ac1481387262493a9ebc00129fe9b0085a6a"},"msg":"[Object: null prototype] {\n  slot: 142191193,\n  id: 'ff3710a432d599f379c38fc110d8ac1481387262493a9ebc00129fe9b0085a6a'\n}","time":"2024-12-09T15:18:10.390Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:10.515Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191193,"id":"ff3710a432d599f379c38fc110d8ac1481387262493a9ebc00129fe9b0085a6a"},"msg":"[Object: null prototype] {\n  slot: 142191193,\n  id: 'ff3710a432d599f379c38fc110d8ac1481387262493a9ebc00129fe9b0085a6a'\n}","time":"2024-12-09T15:18:10.518Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:15.385Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191203,"id":"c259e8406a3e7c921213669b67beca3e2607870548f8428056595220729a1d18"},"msg":"[Object: null prototype] {\n  slot: 142191203,\n  id: 'c259e8406a3e7c921213669b67beca3e2607870548f8428056595220729a1d18'\n}","time":"2024-12-09T15:18:15.390Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:15.515Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191203,"id":"c259e8406a3e7c921213669b67beca3e2607870548f8428056595220729a1d18"},"msg":"[Object: null prototype] {\n  slot: 142191203,\n  id: 'c259e8406a3e7c921213669b67beca3e2607870548f8428056595220729a1d18'\n}","time":"2024-12-09T15:18:15.518Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:20.388Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191207,"id":"285608fa8f5223d249c75060633ad4f27dbe98c5fa80949db0431a66b4c91f74"},"msg":"[Object: null prototype] {\n  slot: 142191207,\n  id: '285608fa8f5223d249c75060633ad4f27dbe98c5fa80949db0431a66b4c91f74'\n}","time":"2024-12-09T15:18:20.395Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:20.516Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191207,"id":"285608fa8f5223d249c75060633ad4f27dbe98c5fa80949db0431a66b4c91f74"},"msg":"[Object: null prototype] {\n  slot: 142191207,\n  id: '285608fa8f5223d249c75060633ad4f27dbe98c5fa80949db0431a66b4c91f74'\n}","time":"2024-12-09T15:18:20.518Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:25.387Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191207,"id":"285608fa8f5223d249c75060633ad4f27dbe98c5fa80949db0431a66b4c91f74"},"msg":"[Object: null prototype] {\n  slot: 142191207,\n  id: '285608fa8f5223d249c75060633ad4f27dbe98c5fa80949db0431a66b4c91f74'\n}","time":"2024-12-09T15:18:25.391Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:25.516Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191207,"id":"285608fa8f5223d249c75060633ad4f27dbe98c5fa80949db0431a66b4c91f74"},"msg":"[Object: null prototype] {\n  slot: 142191207,\n  id: '285608fa8f5223d249c75060633ad4f27dbe98c5fa80949db0431a66b4c91f74'\n}","time":"2024-12-09T15:18:25.518Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:30.386Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191219,"id":"cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8"},"msg":"[Object: null prototype] {\n  slot: 142191219,\n  id: 'cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8'\n}","time":"2024-12-09T15:18:30.390Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:30.515Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191219,"id":"cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8"},"msg":"[Object: null prototype] {\n  slot: 142191219,\n  id: 'cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8'\n}","time":"2024-12-09T15:18:30.518Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:35.387Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191219,"id":"cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8"},"msg":"[Object: null prototype] {\n  slot: 142191219,\n  id: 'cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8'\n}","time":"2024-12-09T15:18:35.391Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:35.516Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191219,"id":"cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8"},"msg":"[Object: null prototype] {\n  slot: 142191219,\n  id: 'cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8'\n}","time":"2024-12-09T15:18:35.518Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:40.387Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191219,"id":"cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8"},"msg":"[Object: null prototype] {\n  slot: 142191219,\n  id: 'cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8'\n}","time":"2024-12-09T15:18:40.392Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","msg":"initialized","time":"2024-12-09T15:18:40.515Z","v":0}
{"name":"cardano-graphql","hostname":"cardano-1-graphql-795c56c74d-qfbsk","pid":1,"level":30,"module":"CardanoNodeClient","tip":{"slot":142191219,"id":"cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8"},"msg":"[Object: null prototype] {\n  slot: 142191219,\n  id: 'cf84f2bbb87772d5e8bebb5618f2db420f7413baa67e284667ce071334ea4ef8'\n}","time":"2024-12-09T15:18:40.517Z","v":0}
```
And for background:
```
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.495Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.513Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.532Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.557Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.580Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.603Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"dfc55fd88deeff37bd4fef24e07d0778fde62ffea5ad8fd3758fa37f534e41494c42524f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.623Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.645Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.663Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.686Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"19fecb7afbc6b92a3c78ad1be2025d8309d2c0cd47f0f8f720b52bce00","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.706Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.719Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"6b1512a22f21f873950300ebcef8d87b8f785acb271cbfdd0149fa4b534153","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.742Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"707e7bc5ff06baefeb5ed3678e796c9d5aa8c7a8dd12faab8f1469b143617264616e6f43726f636f","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.761Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"19fecb7afbc6b92a3c78ad1be2025d8309d2c0cd47f0f8f720b52bce00","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.785Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"19fecb7afbc6b92a3c78ad1be2025d8309d2c0cd47f0f8f720b52bce00","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.801Z","v":0}
{"name":"background","hostname":"cardano-1-graphql-background-7f9cfc8495-wqfcf","pid":1,"level":30,"module":"HasuraClient","assetId":"748f6fcf5f0614d4ce934ac91e2a81ceda312246ccbe0cf7219d38c4535445414c","msg":"Adding metadata to asset","time":"2024-12-09T15:19:53.815Z","v":0}
```

**Comment by petarmitrovic** [comment]
I solved the problem by updating ogmios to 6.9.0, which wasn't actually specified in the [compatibility matrix](https://docs.cardano.org/developer-resources/release-notes/comp-matrix/).
